### PR TITLE
Add parametric endpoint lookup - 

### PR DESCRIPTION
**Pull request by EduardoAquinta**
I have completed the parametric endpoint lookup, with a couple of tests to see it through. I'm not sure if I need to add some more tests or not?

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -3,10 +3,11 @@ const testData = require("../db/data/test-data")
 const request = require('supertest');
 const app = require('../app');
 const db = require('../db/connection');
+const { get } = require("express/lib/response");
```
Comment (emilyb93) [inline __tests__/app.test.js:6]: rogue import

File: app.js [hunk 2]
```diff
@@ -3,24 +3,40 @@ const app = express();
...
 //Error handling for status 404
 app.use("/*", (request, response, next) => {
-    response.status(404).send({ msg: "Route not found" });
+    response.status(404).send({ msg: "page not found" });
 });
 
+//Error handling for unknown error messages
+app.use((error, request, response, next) => {
+    response.status(error.status).send( {msg: error.msg });
+    console.log({msg: error.msg}, "<---Error Message")
```
Comment (emilyb93) [inline app.js:38]: Remove console.logs before merging. If your end point is finished and ready to merge with main, we should not need these for troubleshooting

**Review by emilyb93 — APPROVED** [review]
Just some minor changes but free to merge afterwards

File: __tests__/app.test.js (hunk 1 above)
Comment (EduardoAquinta) [reply __tests__/app.test.js:6]: urgh!

File: app.js (hunk 2 above)
Comment (EduardoAquinta) [reply app.js:38]: I thought this after I pushed the branch... at least you can see I am having lots of trouble and then getting myself out of it ;)

**Comment by EduardoAquinta** [comment]
Have made all the changes re:console.logs, will get rid of the rogue import too!

**Pull request merged**